packages/components/src/vue/components/morph-parts/inflections-list.vue

```diff
@@ -138,15 +170,29 @@
       }
     },
     computed: {
+      listClasses () {
+        const klasses = ["alpheios-inflections-list__inflections"]
+        klasses.push(`alpheios-inflections-list__${this.listclass}`)
+        return klasses.join(" ")
```

Comment:
I'm wondering if the name `klasses` is intentional. Or is it a typo?

Reply:
ah it was just habit to avoid using the word class which is reserved in some languages.

Reply:
will rename this to classNames